### DISCUSSION TITLE
Tokens Tooltip - Display alias value along with real value

### DISCUSSION THREAD

**Discussion by 91volt** [six7/figma-tokens#717] [Q&A]
Currently on tokens preview (tooltip while hovering on tokens) is displayed only the "real/final" value associated and not the alias name. 

<img width="350" alt="Schermata 2022-05-16 alle 16 24 14" src="https://user-images.githubusercontent.com/92443068/168616296-08bc5838-3478-4adc-968b-a544b370686e.png">

This information is currently accessible only on the token's editing pop over, adding friction to its discoverability and readability. 
<img width="455" alt="Schermata 2022-05-16 alle 16 24 23" src="https://user-images.githubusercontent.com/92443068/168617568-16723ff8-29cc-463e-a748-770c81e77cbc.png">

If a token contains an alias reference I would expect to see the alias name along the real value on its preview.
![figma-tokens-enhancement](https://user-images.githubusercontent.com/92443068/168621218-f12861a0-96ba-40ed-a415-1a556f1c39d5.png)

If I remember well this was the behavior on old releases, Is it an unwanted regression introduced in the last update (v99)?

----
Thank you for the efforts on crafting this very useful plugin.

**Comment by six7** [comment]
Thanks for reporting! This was indeed an unintended regression, we'll reintroduce this in the next version.
